homeassistant/components/smartthings/light.py

```diff
@@ -49,9 +49,11 @@ async def async_setup_entry(
 
 
 def convert_scale(
-    value: float, value_scale: int, target_scale: int, round_digits: int = 4
+    value: float | None, value_scale: int, target_scale: int, round_digits: int = 4
 ) -> float:
     """Convert a value to a different scale."""
+    if value is None:
+        return 0.0
```

Comment:
In what condition can value be `None`, and then is it correct to return 0.0. Or should we return `None` is some cases?

Reply:
Okay so I asked some more questions to the reporter, and apprently there are stateless devices that respond to commands, but just don't expose states. While this PR doesn't fix that, it does make the integration stop from raising errors. I will check with SmartThings how we can do this properly, but until then, I think we should stop the code for raising errors

Reply:
Can we add a note to explain this?

Reply:
Normally we return `None` which will be converted to `"unknown"` by the base entity when an entity has a measurement that is `None` or not known.

Reply:
That's correct, I forgot about that. Thanks!